qsimcirq/qsim_simulator.py

```diff
@@ -526,8 +497,70 @@ def simulate_sweep_iter(
             assert qsim_state.dtype == np.float32
             assert qsim_state.ndim == 1
 
+            yield prs, qsim_state.view(np.complex64), cirq_order
+
+    def simulate_into_1d_array(
+        self,
+        program: cirq.AbstractCircuit,
+        param_resolver: cirq.ParamResolverOrSimilarType = None,
+        qubit_order: cirq.QubitOrderOrList = cirq.ops.QubitOrder.DEFAULT,
+        initial_state: Any = None,
+    ) -> Tuple[cirq.ParamResolver, np.ndarray, Sequence[int]]:
+        """Same as simulate() but returns raw simulation result without wrapping it.
+
+            The returned result is not wrapped in a StateVectorTrialResult but can be used
+            to create a StateVectorTrialResult.
+
+        Retruns:
+            A parameter resolver.
+            Final simulation state vector as a 1D array.
+            Qubit order.
```

Comment:
`Returns` should describe the (single) object returned. Paraphrasing:
```
Returns:
    Tuple of (param resolver, final state, qubit order)
```

Reply:
done